src/frontend/apps/standalone_site/src/utils/lazyImport.ts

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
```

Comment:
```suggestion
import React from 'react';
```

Reply:
Importing React is no longer required since React 17 and eslint is configured to ignore it.